java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -89,6 +89,34 @@ public void ping_with_message(BaseClient client) {
         assertEquals("H3LL0", data);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void unlink_multiple_keys(BaseClient client) {
```

Comment:
I see we dont have a test for unlink in node, we need to fix it, but add one here as well

I suggest looking at both node & python wrappers